applications/.depcheckrc

```diff
@@ -0,0 +1,21 @@
+ignores: [
+  # common
+  "ts-loader",
+  "css-loader",
+  "html-loader",
+  "sass-loader",
+  "sass",
+  "@types/jest",
+  "@jest/globals",
+  "ts-jest",
+  "ts-node",  # TODO: Check this one, if we need the package at all
+  "tslib",    # TODO: Check if the package is required
```

Comment:
Unfortunately it is needed by angular and svelte
ref: https://github.com/kfarkasHU/wraithlight/pull/26

would be awesome to add a comment to this, or move it to two separate groups